README.md

```diff
@@ -674,6 +674,13 @@ bat() {
 
 ## Troubleshooting
 
+### Garbled output
+
+If an input file contains color codes or other ANSI escape sequences, `bat` will have problems
+performing syntax highlighting and text wrapping and, thus the output can become garbled.
```

Comment:
Comma placements seems a bit weird? Maybe change to

```suggestion
performing syntax highlighting and text wrapping, and thus the output can become garbled.
```

Otherwise it looks fine to me.

Reply:
done :)